### PR TITLE
Add named params in Controllers

### PR DESCRIPTION
**Pull request by olivierbon**
Not sure if that's needed but just in case :)

### CODE REVIEW

File: docs/3.x/routing.md [hunk 1]
```diff
@@ -148,3 +148,22 @@ If you access `http://my-project.test/blog/archive/2018`, your `blog/_archive.tw
...
+
+To access named parameters in your contollers, you will need to add the parameter(s) to your controller's action definition.
+
+For example, with this URL rule:
+
+```php
+'blog/archive/<year:\d{4}>' => 'controller/action/foo',
+```
+
+You controller method would be:
```
Comment (mattstein) [inline docs/3.x/routing.md]: Should be “_Your_ controller method” but otherwise good to go!

**Review by mattstein — CHANGES_REQUESTED** [review]
Thanks for this @olivierbon! This is a nice place for that extra bit of clarity, just one typo to fix. (I’m not sure how reviews should work so I hope I did this right.)

File: docs/3.x/routing.md (hunk 1 above)
Comment (olivierbon) [reply docs/3.x/routing.md]: I'm just going to go hide now.

Comment (mattstein) [reply docs/3.x/routing.md]: Oh I didn’t mean to be rude—I’ll take care of it. Again, thank you!

**Review by mattstein — APPROVED** [review]

**Pull request merged**